test/fileio.jl

```diff
@@ -36,20 +36,41 @@ end
 
 let fn = tempname() * ".h5"
   h5open(fn, "w"; track_order=true) do io
-    fcpl = HDF5.get_create_properties(io)
-    @test fcpl.track_order
+    @test HDF5.get_create_properties(io).track_order
```

Comment:
is this a NFC (non functional change)?

Reply:
Rationale: consistency with the test written after. But I won't argue over a 2 lines change. Restored.